frontend/src/AppBuilder/Widgets/Table/Table.jsx

```diff
@@ -974,12 +968,6 @@ export const Table = React.memo(
       });
     }, [JSON.stringify(tableDetails.selectedRowsDetails)]);
 
-    const pageData = page.map((row) => row.original);
```

Comment:
Removed this ussEffect(duplicate) as this is already being done in another UseEffect.